ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -92,6 +99,192 @@ protected override BorderedCanvas CreateViewInstance(ThemedReactContext reactCon
             return new BorderedCanvas();
         }
 
+        #region drag and drop
+#if WINDOWS_UWP
```

Comment:
please make this emit an NotImplemented Red/Yellow Box error on non-UWP platforms

Reply:
Done